model.js

```diff
@@ -18,8 +18,8 @@ const modelHeader = tcomb.struct(
 const modelItem = tcomb.struct(
     {
         description: tcomb.String,
-        qty: tcomb.Integer,
-        value: tcomb.Number
+        qty: tcomb.Any,
```

Comment:
What do you think instrad Any use Number on both?, number support decimal

Reply:
It has been almost a year, and I can't remember why I didn't use number instead of Any, but I'm sure there was a reason. xD